portal-api/src/v1/controllers/facilities.controller.js

```diff
@@ -128,33 +116,30 @@ const queryAllFacilities = async (
       {
         $sort: {
           ...sort,
-          updatedAt: -1,
+          updatedAt: -1, // TODO SR-8: Sort order -- no $eq expression needed
```

Comment:
Remove todo once noted:
```suggestion
          updatedAt: -1,
```